addons/mass_mailing_crm/demo/mailing_mailing.xml

```diff
@@ -14,7 +14,7 @@
         <field name="reply_to_mode">new</field>
         <field name="reply_to">{{ object.company_id.email }}</field>
         <field name="body_arch" type="html">
-<div class="oe_structure o_editable">
+<div class="oe_structure o_mail_wrapper_td">
```

Comment:
This template was broken before;
- Go on this template in edit mode
- Add a snippet on it and save.
- Reload the page.

-> It is impossible to add a snippet